### ISSUE TITLE
Language Grammar - row-type requires at least one field

### ISSUE THREAD

**Issue by bgribaudo** [MicrosoftDocs/powerquery-docs#36]
In the language grammar, [Type expression](https://docs.microsoft.com/en-us/powerquery-m/m-spec-consolidated-grammar#type-expression)’s definition of `row-type` requires that at least one field be specified. This disallows the following, which is  valid Power Query syntax, per testing in Excel: 
````
type table []
````

To rectify, I believe `row-type`'s definition should be changed so that the field specification list is optional--something like:
```
row-type:
      [ field-specification-list(opt) ]
````

Can you confirm that this is the correct definition?

Thank you!

**Comment by bgribaudo** [comment]
Related but different: #19

**Comment by JasonWHowell** [comment]
Please ignore (I'm using bot automation to add labels). #please-open

**Comment by JasonWHowell** [comment]
This issue is pending fix in pull request #15 which is queued for review.  Thanks @bgribaudo for the help 
#please-open
